### PR TITLE
Added filter-x icon

### PR DESCRIPTION
**Pull request by gubser**
Hi 👋

I needed an icon to remove a filter, so I used the x from `screen-share-off` and cut the polygon of `filter`.

image

### CODE REVIEW

**Comment by ericfennis** [comment]
@gubser Nice icon! 
Maybe we should call this `filter-x` instead of `filter-off` 🤔

**Comment by gubser** [comment]
Sure, here you are :+1:

File: icons/filter-x.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="m13.013 3h-11.013l8 9.46v6.54l4 2v-8.54l0.89973-1.0554"/>
+  <path d="m22 3-5 5"/>
+  <path d="m17 3 5 5"/>
```
Comment (ericfennis) [inline icons/filter-x.svg:14]: Formatting
```suggestion
  <path d="m13.013 3h-11.013l8 9.46v6.54l4 2v-8.54l0.89973-1.0554" />
  <path d="m22 3-5 5" />
  <path d="m17 3 5 5" />
```

**Review by ericfennis — APPROVED** [review]
Awesome @gubser ! 
Good to go.

**Pull request merged**